### DISCUSSION TITLE
Seeking conda-forge recipe maintainers for m2w64 toolchain packages

### DISCUSSION THREAD

**Discussion by mwtoews** [msys2/MINGW-packages#12013] [General]
Since December 2016, the msys2 toolchain packages (`m2w64-gcc`, `m2w64-gcc-fortran`, etc.) have not been maintained and is [now archived](https://github.com/conda-archive/conda-recipes/blob/master/msys2/m2w64-toolchain/meta.yaml). These tools and compilers have been frozen in time at GCC version 5.3.0 (released December 2015). This means many conda packages built for Windows depend on really old compilers.

Issues like [this one](https://github.com/conda-forge/toolchain-feedstock/issues/60) have been raised to try and update the packages, but no knowledgeable volunteers have yet stepped forward to take lead.

This is a huge request on behalf of Windows conda forge users and other maintainers to see if there are any volunteers willing to help revitalize the msys2 toolchain packages with modern conda recipes to bring up-to-date tools and compilers to conda communities. The conda-forge repo in question is at https://github.com/conda-forge/msys2-recipes/tree/master/msys2

**Comment by mwtoews** [comment]
I've just investigated the previous maintainer of msys2 on conda-forge, and I see it was co-creator of MSYS2 @mingwandroid who was fighting cancer since March 2018, and sadly passed away in April 2021. Thanks Ray! So I see the request in this discussion is to fill some pretty big shoes.

**Comment by naveen521kk** [comment]
Is it as simple as just updating the links as when those packages get updated here? Or the PKGBUILD's here needs to be converted into conda's build script (ie. build from source)? If it's the latter, I think it will be hard to convince anybody here to do that.

**Reply by msarahan** [reply]
Should be more the former. This looks like a good job for the bot, if we were able to script/automate porting these things into feedstocks, the bot could probably take over and keep things better-maintained.

**Reply by naveen521kk** [reply]
> Should be more the former. This looks like a good job for the bot, if we were able to script/automate porting these things into feedstocks, the bot could probably take over and keep things better-maintained.

Yeah, making a bot do this is great. I think one can get the URLs of the recent package by parsing the [Pacman database file](https://repo.msys2.org/mingw/mingw64/mingw64.db) (there is a library for doing that, [pacdb](https://pypi.org/project/pacdb)).
